File: Course/ITP1/2_B/main.py
#
# AOJ ITP1 / 2_B - Range
#
# https://onlinejudge.u-aizu.ac.jp/courses/lesson/2/ITP1/all/ITP1_2_B

# ---------------------------------------------------------------------------- #

def main():
    # [ input ]
    a, b, c = map(int, input().split())

    # [ output ]
    if (a < b < c):
        print("Yes")
    else:
        print("No")

    # [ return ]
    return

# ---------------------------------------------------------------------------- #

if __name__ == '__main__':
    main()

File: Course/ITP1/2_B/main.cpp
//
// AOJ ITP1 /2_B - Range
//
// https://onlinejudge.u-aizu.ac.jp/courses/lesson/2/ITP1/all/ITP1_2_B


/* -------------------------------------------------------------------------- */

#include <bits/stdc++.h>
using namespace std;

#define rep(i, a, b) for (int i = (a); i < (b); i++)
using ll = long long;

int main()
{
    // [ input ]
    int a, b, c;
    cin >> a >> b >> c;

    // [ output ]
    if (a < b && b < c)
        cout << "Yes" << endl;
    else
        cout << "No" << endl;

    // [ return ]
    return (0);
}
